capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -11,5 +11,5 @@
    * Take a list of suggestion data and returns a list of Suggestions objects to the caller. This
    * object contains all data relevant to the Search feature.
    */
-  List<SearchData> getSearchSuggestions(List<SuggestionData> collegeNames);
+  List<SearchData> getSearchSuggestions(List<String> collegeNames);
```

Comment:
How should the client supply the `List<String>` in this method signature? The client just has free text from the autocomplete box. How (or should?) will the client convert that into a list?

Reply:
client shouldn't be converting it into a list :) just need a single String!